sound/soc/sof/intel/hda.c

```diff
@@ -865,6 +897,10 @@ static int hda_init_caps(struct snd_sof_dev *sdev)
 
 	hda_bus_ml_get_capabilities(bus);
 
+	/* Skip SoundWire if is not supported */
```

Comment:
typo if "it" is not supported or just if not supported

Reply:
OK, thanks